app/hooks/useSidebar.ts

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
Need to rename this hook, it doesn't need to have anything to do with the sidebar 😆 – just tracking how many context menus are open.